package.json

```
@@ -91,15 +91,18 @@
     "**/@oclif/command": "<=1.8.3",
```

```suggestion
    "**/@npmcli/fs": "<1.1.0",
    "**/@oclif/command": "<=1.8.3",
```
This will fix the failing node 10 build.